spec/unit/provider/oneview_logical_interconnect_spec.rb

```diff
@@ -74,6 +75,7 @@
     end
 
     it 'return false when the resource does not exists' do
+      allow_any_instance_of(resource_type).to receive(:retrieve!).and_return(true)
```

Comment:
Its a duplication as you already added this line to before(:each) block that will run before all the test cases under the context.
Applicable for line number 78, 88, 115